Compatibility/runtimeconfig.template.json

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// runtimeconfig.template.json replaces App.Config based AppContext configuration
+// The following AppContext based flags are available for use in WPF Apps, and are initialized with 
+// Default values as shown below
+{
+  "configProperties": {
+    /*
+    // BuildTasksAppContextSwitches
+
+    // The WPF MarkupCompiler provides compilation services for XAML markup files. In the .NET Framework 4.7.1
+    // and earlier versions, the default hash algorithm used for checksums was SHA1. Due to recent security
+    // concerns with SHA1, this default has been changed to SHA256 starting with the .NET Framework 4.7.2.
+    // This change affects all checksum generation for markup files during compilation.
+    "Switch.System.Windows.Markup.DoNotUseSha256ForMarkupCompilerChecksumAlgorithm": false,
+
+
+    // CoreAppContextSwitches
+
+    // Determines whether DPI changes occur on a per-system (a value of false) or per-monitor basis (a value of true).
+    "Switch.System.Windows.DoNotScaleForDpiChanges": false,
+
+    // Switch to enable the correct exception being thrown in ImageSourceConverter.ConvertFrom instead of NullReferenceException
+    "Switch.System.Windows.Media.ImageSourceConverter.OverrideExceptionWithNullReferenceException": false,
+
+    // When set to true, WPF will not enable the compatibility breaking bug fixes associated with
+    // features advertised by ""Switch.System.Windows.PresentationDpiCapabilityTier2""
+    // The following behavior would be turned off when this flag is set by the application:
+    // - Improvements to how HwndHost sizes child windows in response to DPI changes
+    // - Improvements to window placement during startup
+    // The following fixes would remain unaffected:
+    //  - High-DPI related accessibility fixes.
+    "Switch.System.Windows.DoNotUsePresentationDpiCapabilityTier2OrGreater": false,
+
+    // Switch to enable WPF support for the WM_POINTER based stylus/touch stack
+    "Switch.System.Windows.Input.Stylus.DisableStylusAndTouchSupport": false,
+
+    // https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/mitigation-pointer-based-touch-and-stylus-support
+    "Switch.System.Windows.Input.Stylus.EnablePointerSupport": false,
```

Comment:
This is the switch you probably meant for the comment about enabling WM_POINTER.